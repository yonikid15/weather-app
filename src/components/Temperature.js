import React from 'react';

const Temperature = ({ temperature }) => {
  return (
    <div>
      <h3>
          <span>Temperature: </span>
          <span>{Math.round(temperature - 274)}&#8451;</span>
      </h3>
    </div>
  );
};

export default Temperature;