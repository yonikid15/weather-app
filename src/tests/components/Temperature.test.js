import React from 'react';
import { shallow } from 'enzyme';
import Temperature from '../../components/Temperature';

test('should render Temperature correctly', () => {
  const wrapper = shallow( <Temperature temperature={300} /> );
  expect( wrapper ).toMatchSnapshot();
});