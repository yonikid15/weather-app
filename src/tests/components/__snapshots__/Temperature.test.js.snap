// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Temperature correctly 1`] = `
<div
  className="temperature"
>
  <h3
    className="temperature__title"
  >
    <i
      className="fas fa-thermometer-half temperature__icon"
    />
    <span>
      26
      Â°C
    </span>
  </h3>
</div>
`;
